theatre/studio/src/panels/DetailPanel/prop-editors/simple-editors/StringLiteralPropEditor.tsx

```diff
@@ -0,0 +1,34 @@
+import type {PropTypeConfig_StringLiteral} from '@theatre/core/propTypes'
```

Comment:
al these files in `simple-editors` should be marked with `git rename`. Skipping review on them for now.